src/plugins/condition/utils/operations.js

```diff
@@ -226,7 +228,9 @@ export const OPERATIONS = [
     {
         name: 'enumValueIsNot',
         operation: function (input) {
-            return input[0] !== input[1];
+            let stringInputs = [];
+            input.forEach(inputValue => stringInputs.push(inputValue !== undefined ? inputValue.toString() : ''));
```

Comment:
may be create new function, as it is used in other place as well.

Reply:
Good point. Will do.